plugins/node/opentelemetry-instrumentation-redis-4/README.md

```diff
@@ -53,9 +53,8 @@ Redis instrumentation has a few options available to choose from. You can set th
 
 #### Custom `db.statement` Serializer
 
-The instrumentation serializes the command into a Span attribute called
-`db.statement`. The default serialization sets the attribute to the command
-name, without the command arguments.
+The instrumentation serializes the command into a Span attribute called `db.statement`. The standard serialization format attempts to be as informative as possible while avoiding the export of potentially sensitive data. The number of serialized arguments depends on the specific command, see the configuration
+list in `packages/opentelemetry-redis-common/src/index.ts`.
 
 It is also possible to define a custom serialization function. The function
 will receive the command name and arguments and must return a string.
```

Comment:
The text below says: "Here is a simple example to serialize the command name and arguments:"

now that the default serializer also captures command arguments, it might be worth rephrasing to something like "Here is a simple example to serialize the command name and all command arguments. Notice that it might capture sensitive data and big payloads".

Reply:
I'll it also in redis, but not in ioredis (there is another code example there)